lib/Stan_math_code_gen.ml

```diff
@@ -183,19 +257,16 @@ let rec pp_statement ppf {stmt; sloc} =
       let argtypetemplates =
```

Comment:
You use `ignore decl_adtype` above. Should that be our preferred style rather than just matching on `Decl {decl_id; decl_type; _}` ?

Reply:
Not sure.